RPIPs/RPIP-47.md

```diff
@@ -31,10 +31,10 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 - Each `Megapool delegate` contract SHALL have an execution layer `expiration_block` variable which should initialize to "no expiration"
 - Each `Megapool delegate` contract SHALL have a non-modifiable `upgrade_buffer` value
   - The `Megapool delegate`'s `upgrade_buffer` SHALL be 120 days
-- The only permitted modification to a `Megapool delegate`'s `expiration_block` variable SHALL be for any protocol contract to set its value to now + its `upgrade_buffer`
+- The only permitted modification to a `Megapool delegate`'s `expiration_block` variable SHALL be for any protocol contract to set its value to now<-[is "now" ambiguous? I'm not sure how else you would explain this point though]  + its `upgrade_buffer`
```

Comment:
'the current block' is the probably the best alternative.

Reply:
or even "the current block"; worth a tweak

Reply:
I'll go with this unless I hear otherwise:
> The only permitted modification to a `Megapool delegate`'s `expiration_block` variable SHALL be for any protocol contract to set its value to the current block.

Reply:
That's wrong? Should be

> The only permitted modification to a `Megapool delegate`'s `expiration_block` variable SHALL be for any protocol contract to set its value to the current block + its `upgrade_buffer`.

Reply:
Yes, needs to be current + buffer.

Reply:
oops, didn't realize my committed comment was mid-sentence... I'll include that in polish PR

Reply:
:rocket: